frontend/src/components/Pages/InspectionPage/ScheduleMissionDialogs.tsx

```diff
@@ -110,10 +110,12 @@ export const ScheduleMissionDialog = (props: IProps): JSX.Element => {
         if (!selectedRobot || !missionsToSchedule) return
 
         missionsToSchedule.forEach((mission) => {
```

Comment:
Should there also be a frontend check such that the user get feedback before going trough the localization dialog. Like a check on line 101 where the user get a dialog or something when the pressure is low,  instead of actually scheduling?

Reply:
This has been added, but I would like your feedback on it